src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -954,7 +961,7 @@ EclipseGrid::EclipseGrid(const Deck& deck, const int * actnum)
         const std::vector<double>& drv     = deck.getKeyword<ParserKeywords::DRV>().getSIDoubleData();
         const std::vector<double>& dthetav = deck.getKeyword<ParserKeywords::DTHETAV>().getSIDoubleData();
         const std::vector<double>& tops    = deck.getKeyword<ParserKeywords::TOPS>().getSIDoubleData();
-        OpmLog::info(fmt::format("\nCreating spiderweb grid from keywords DRV, DTHETAV, DZV and TOPS"));
+        OpmLog::info(fmt::format("\nCreating " + kind + " grid from keywords DRV, DTHETAV, DZV and TOPS"));
```

Comment:
Hehe - when you are actually using `fmtlib` you should ehhh well do it:
```C++
OpmLog::info(fmt::format("\nCreating {} grid from keywords DRV, DTHETAV, DZV and TOPS", kind));
```

Reply:
:smile: you can tell I am rusty in my C++, no? I think I did not even notice the format that was there already... will fix.